README.md

```diff
@@ -154,7 +181,7 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+60 valid sentences
```

Comment:
# 0

Because you can have any number of adjectives, there are an infinite number.